common/setups/hybrid/gmm_system.py

```diff
@@ -204,7 +224,7 @@ def monophone_training(
     def cart_and_lda(
         self,
         name: str,
-        corpus: str,
+        corpus_key: str,
         initial_flow: str,
```

Comment:
Here we also have the `object` vs `str` issue, as those are not real flows but flow keys. We should also rename this...